cairo/tests/test_serde.py

```diff
@@ -128,7 +144,10 @@ class TestSerde:
     @given(b=...)
     # 20 examples per type
     # Cannot build a type object from the dict until we upgrade to python 3.12
-    @settings(max_examples=20 * len(_cairo_struct_to_python_type))
+    @settings(
+        max_examples=20 * len(_cairo_struct_to_python_type),
+        suppress_health_check=[HealthCheck.data_too_large],
```

Comment:
the new additions will generate consequent amounts of data which are quite slow

Reply:
since tests are cached, it's not a big deal I guess